_episodes/00-sql-introduction.md

```diff
@@ -179,7 +179,7 @@ database.  We'll be creating this database from the three `csv` files
 we downloaded earlier.  Close the currently open database and then 
 follow these instructions: 
 
-1. Start a New Database **Database -> New Database**
+1. Start a New Database **Database -> New Database**. Give a name **Ok -> Open**. No selection needed.
```

Comment:
@orchid00  does this look good.

1. Start a New Database 
    - **Database -> New Database**. 
    - Give a name **Ok -> Open**. Creates the database in that opened folder
2. Start the import **Database -> Import**
3. Select the `surveys.csv` file to import
4. Give the table a name that matches the file name (`surveys`), or use the default
5. If the first row has column headings, check the appropriate box
6. Make sure the delimiter and quotation options are appropriate for the CSV files.  Ensure 'Ignore trailing Separator/Delimiter' is left *unchecked*.
7. Press **OK**
8. When asked if you want to modify the table, click **OK**
9. Set the data types for each field using the suggestions in the table below (this includes fields from `plots` and `species` tables also):